pkg/symbol/addr2line/go.go

```diff
@@ -89,6 +91,9 @@ func (gl *GoLiner) PCToLines(addr uint64) (lines []profile.LocationLine, err err
 
 	name := "?"
 	// TODO(kakkoyun): Do we need to consider the base address for any part of Go binaries?
```

Comment:
I think now this comment is deprecated, no?

Reply:
Removing it... I wasn't sure about this TODO comment and left it for review :).